views/postgres/networking.erb

```diff
@@ -1,68 +1,17 @@
-<% has_firewall = @pg.customer_firewall %>
-
 <div class="p-6">
   <div class="md:flex md:items-center md:justify-between pb-2 lg:pb-4">
     <div class="min-w-0 flex-1">
       <h3 class="text-2xl font-bold leading-7 text-gray-900 sm:truncate sm:text-2xl sm:tracking-tight">Firewall Rules</h3>
-      <% unless has_firewall %>
-        <p class="text-gray-500 mt-2">PostgreSQL firewall was deleted, manage firewall rules using an appropriate firewall on the <a class="text-orange-600" href="<%= path(@pg.private_subnet) %>/networking">private subnet</a>.</p>
+      <p class="text-gray-500 mt-2">
+      <% if firewall = @pg.customer_firewall %>
+          You can 
+          <a class="text-orange-600" href="<%= path(firewall) %>/networking">manage firewall rules using the firewall for this PostgreSQL database</a>.
+      <% else %>
+          The firewall related to this PostgreSQL database was deleted or detached from the related private subnet,
+          you can manage firewall rules using an appropriate firewall on the
+          <a class="text-orange-600" href="<%= path(@pg.private_subnet) %>/networking">private subnet</a>.
```

Comment:
since we are removing the pg specific firewalls from the UI. For the sake of UX, we might add some predefined port selection in the firewall creation page? Like, the port box stays edittable but we can pick a "Postgres" type that fills 5432 automatically, "pgbouncer", "ssh", "http", or "https". What do you think? It is not really something we have to solve in this PR but it might be a good addition later to increase usability.